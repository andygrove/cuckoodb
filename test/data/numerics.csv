a,b,a_f,b_f
2,3,3.14,-2.13
5,5,3.0,3
0,2,2.1,4.00