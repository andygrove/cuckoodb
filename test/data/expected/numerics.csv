5,-1,6,0,1.01,5.2700005,-6.6882005,-1.4741783
10,0,25,1,6,0,9,1
2,-2,0,0,6.1,-1.9000001,8.4,0.525
